modules/loader.js

```diff
@@ -626,42 +577,13 @@ Loader.prototype.getNetwork = function (cb) {
 	if (__private.network.height > 0 && Math.abs(__private.network.height - modules.blocks.getLastBlock().height) === 1) {
 		return setImmediate(cb, null, __private.network);
 	}
-	async.waterfall([
-		function (waterCb) {
-			modules.transport.getFromRandomPeer({
-				api: '/list',
-				method: 'GET'
-			}, function (err, res) {
-				if (err) {
-					return setImmediate(waterCb, err);
-				} else {
-					return setImmediate(waterCb, null, res);
-				}
-			});
-		},
-		function (res, waterCb) {
-			library.schema.validate(res.body, schema.getNetwork.peers, function (err) {
-				var peers = modules.peers.acceptable(res.body.peers);
 
-				if (err) {
-					return setImmediate(waterCb, err);
-				} else {
-					library.logger.log(['Received', peers.length, 'peers from'].join(' '), res.peer.string);
-					return setImmediate(waterCb, null, peers);
-				}
-			});
-		},
-		function (peers, waterCb) {
-			async.map(peers, __private.getPeer, function (err, peers) {
-				return setImmediate(waterCb, err, peers);
-			});
-		}
-	], function (err, heights) {
+	modules.peers.list({}, function (err, peers) {
```

Comment:
Instead of using `modules.peers.list` we can use also `library.logic.peers.list` (without normalization) - maybe that is better?